.github/workflows/release.yml

```diff
@@ -8,8 +8,13 @@ on:
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
+permissions: {}
 jobs:
   release:
+    permissions:
+      contents: write # to create release 
+      pull-requests: write # to create pull request 
```

Comment:
are you a Changeset user? did you have a chance to confirm that those are the only ones needed for this action?

Reply:
It is optional. I added it prevent the case when a new job is added and it inherits the default permissions, but as long as there is only one job it will use it's own permissions.

Reply:
I didn't have a chance to try it. If it fails it can be expanded.